### PR TITLE
Connect to lambdas

### LINKED ISSUE #14: Send to infraction lambda
https://github.com/Cyberspace-Citizenship-Coefficient/Chrome-Plugin/blob/bf93e5ecbd2cbdd71c60ca84536e985745754b7e/src/rightClick.js#L35

### LINKED ISSUE #15: Load from lambda
https://github.com/Cyberspace-Citizenship-Coefficient/Chrome-Plugin/blob/bf93e5ecbd2cbdd71c60ca84536e985745754b7e/src/rightClick.js#L61

Follow up on https://github.com/Cyberspace-Citizenship-Coefficient/AWS-Lambdas/issues/20

### LINKED ISSUE #16: Get instance ID from lambda
https://github.com/Cyberspace-Citizenship-Coefficient/Chrome-Plugin/blob/bf93e5ecbd2cbdd71c60ca84536e985745754b7e/src/rightClick.js#L3

https://github.com/Cyberspace-Citizenship-Coefficient/AWS-Lambdas/issues/21

### PR DESCRIPTION
**Pull request by TheENGR**

### CODE REVIEW

**Comment by TheENGR** [comment]
Relies on https://github.com/Cyberspace-Citizenship-Coefficient/AWS-Lambdas/issues/19 and https://github.com/Cyberspace-Citizenship-Coefficient/Wall-of-Shame/issues/3

**Comment by TheENGR** [comment]
https://github.com/Cyberspace-Citizenship-Coefficient/AWS-Lambdas/pull/31

File: src/rightClick.js [hunk 1]
```diff
@@ -1,8 +1,23 @@
...
-	// TODO: GET INSTANCE ID FROM LAMBDA
-	chrome.storage.local.set({ instanceID: "xxxxxxxx-xxxx-Mxxx-Nxxx-xxxxxxxxxxxx" });
-	mapMenus();
+	fetch("https://439r656kxf.execute-api.us-east-2.amazonaws.com/dev/device",
+		{
+			body: JSON.stringify({}),
+			method: 'PUT',
+			headers: {'Content-Type': 'application/json'}
+		}
+	)
```
Comment (TheENGR) [inline src/rightClick.js]: @ajaxx 
PUT to device

File: src/rightClick.js [hunk 2]
```diff
@@ -1,8 +1,23 @@
 // Register the Plugin when installed
 chrome.runtime.onInstalled.addListener(() => {
-	// TODO: GET INSTANCE ID FROM LAMBDA
-	chrome.storage.local.set({ instanceID: "xxxxxxxx-xxxx-Mxxx-Nxxx-xxxxxxxxxxxx" });
-	mapMenus();
+	fetch("https://439r656kxf.execute-api.us-east-2.amazonaws.com/dev/device",
```
Comment (ajaxx) [inline src/rightClick.js]: Food for thought.  In a future version, let's move to a function that determines the host.  That way we only need to change the URL in one place.

**Review by ajaxx — DISMISSED** [review]

**Comment by TheENGR** [comment]
image

**Review by appasalvi — APPROVED** [review]

**Pull request merged**